src/main/java/uk/gov/hmcts/reform/blobrouter/model/out/EnvelopeSummaryResponse.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.blobrouter.model.out;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+public class EnvelopeSummaryResponse {
```

Comment:
This class doesn't represent a response, but just an item in the list. Could it be renamed to sth like `EnvelopeSummaryItem`?